src/main/java/com/academy/terai/Controller/AccountResource.java

```diff
@@ -0,0 +1,54 @@
+package com.academy.terai.Controller;
+
+import com.academy.terai.Model.Account;
+import com.academy.terai.Repository.AccountRepository;
+import com.academy.terai.service.AccountService;
+import javassist.NotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.management.openmbean.KeyAlreadyExistsException;
+import java.util.List;
+
+@RestController
+@RequestMapping("/accounts")
+public class AccountResource {
```

Comment:
Controller should be named so, so, a proper name for this class would be 'AccountController'